geojson_length/geojson_length.py

```diff
@@ -24,8 +24,8 @@ def distance_between_two_points(
     """
 
     # GeoPy needs lat, lon
-    point1 = tuple(reversed(point1))
-    point2 = tuple(reversed(point2))
+    point1 = tuple(reversed(point1[0:2]))
+    point2 = tuple(reversed(point2[0:2]))
```

Comment:
Maybe a little bit more "pythonic" solution

```suggestion
    point1 = tuple(reversed(point1[:2]))
    point2 = tuple(reversed(point2[:2]))
```